README.md

```diff
@@ -33,7 +33,7 @@ To further configure your plugin, use the following custom parameters in your `s
 | `logLevel`           | The log level, set to `DEBUG` for extended logging. Defaults to `info`.                                                                                                                                                                                                                                                                                                                           |
 | `enableXrayTracing`  | Set `true` to enable X-Ray tracing on the Lambda functions and API Gateway integrations. Defaults to `false`.                                                                                                                                                                                                                                                                                   |
 | `enableDDTracing`    | Enable Datadog tracing on the Lambda function. Defaults to `true`.                                                                                                                                                                                                                                                                        |
-| `forwarderArn`          | Setting this parameter subscribes the given Datadog forwarder to the Lambda functions’ CloudWatch log groups. Required when `enableDDTracing` is set to `true` unless the subscription is otherwise applied. For example, if a Datadog Forwarder subscription is applied via Datadog's AWS Integration, then `forwarderArn` is not required. Do not set `addExtension` to `true` when using `forwarderArn`.                                                                                                                                                                                                                 |
+| `forwarderArn`          | Setting this parameter subscribes the given Datadog forwarder to the Lambda functions’ CloudWatch log groups. Required when `enableDDTracing` is set to `true` unless the subscription is otherwise applied. For example, if a Datadog Forwarder subscription is applied via Datadog's AWS Integration, then `forwarderArn` is not required. Do not set `addExtension` to `true` when using `forwarderArn`. Note, [CloudFormation pseudo parameters](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html), such as `#{AWS::AccountId}` can NOT be used in this parameter.                                                                                                                                                                                                                |
```

Comment:
```suggestion
| `forwarderArn`          | Setting this parameter subscribes the given Datadog forwarder to the Lambda functions’ CloudWatch log groups. Required when `enableDDTracing` is set to `true` unless the subscription is otherwise applied. For example, if a Datadog Forwarder subscription is applied via Datadog's AWS Integration, then `forwarderArn` is not required. Do not set `addExtension` to `true` when using `forwarderArn`. Note that [CloudFormation pseudo parameters](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html), such as `#{AWS::AccountId}` cannot be used in this parameter.                                                                                                                                                                                                                |
```